### PR TITLE
chore: add Subhojit Dey

### PR DESCRIPTION
**Pull request by Subhojit-Dey1234**
I have added  Subhojit-Dey1234.json in the public/data folder. 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/896"></a>

### CODE REVIEW

File: public/data/Subhojit-Dey1234.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "milestones": [
+      {
+        "title": "Starting Open Source",
+        "date": "January 2022",
+        "icon": "globe",
+        "color": "grey",
+        "description": "My main goal is to progress in Open Source and make some contribution"
+      }
+    ]
+  }
```
Comment (syedsohan) [inline public/data/Subhojit-Dey1234.json:32]:
```suggestion
}

```

File: public/data/Subhojit-Dey1234.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/thesubhojitdey",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "www.linkedin.com/in/subhojit-dey-09",
+        "icon": "LinkedIn"
```
Comment (syedsohan) [inline public/data/Subhojit-Dey1234.json:20]:
```suggestion
        "icon": "linkedin"
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

**Review by syedsohan — CHANGES_REQUESTED** [review]
Change the PR title to - `fix: add Subhojit Dey`

File: public/data/Subhojit-Dey1234.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        "icon": "linkedin"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Starting Open Source",
+        "date": "January 2022",
+        "icon": "globe",
+        "color": "grey",
+        "description": "My main goal is to progress in Open Source and make some contribution"
```
Comment (Panquesito7) [inline public/data/Subhojit-Dey1234.json:29]:
```suggestion
        "description": "My main goal is to progress in Open Source and make some contributions"
```

Comment (krishguptadev) [inline public/data/Subhojit-Dey1234.json:29]:
```suggestion
        "description": "My main goal is to progress in Open Source and make some contribution!"
```

**Review by syedsohan — APPROVED** [review]

**Review by emmalearnscode — APPROVED** [review]

**Review by krishguptadev — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**